src/styles/block/card.scss

```
@@ -0,0 +1,99 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition-duration: .3s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```

---

Not fixed